cypress/e2e/class_management/showGroupsAndClassesInTable.feature

```diff
@@ -1,5 +1,5 @@
 #@release
-@unstable_test
+@stable_test
 Feature: Group - To show groups and classes in one table with respective functionality
 
```

Comment:
In this feature file I can see in the last scenario you are logged in as a `teacher` in `default` instead of `nbc`

``` 
Scenario: As a teacher i can not see the new class page, when the feature flag is off
    Given I am logged in as a 'teacher1_dbc' at 'default'
    When I go to administration page
```

Reply:
that is correct. The scenario here is to test, that the feature flag is working as intended and everything is wrapped in it correctly. The easiest way to do this was just to use another instance, where the feature flag was not activated :)